contrib/builder/deb/generate.sh

```diff
@@ -58,6 +58,8 @@ for version in "${versions[@]}"; do
 		libdevmapper-dev # for "libdevmapper.h"
 		libltdl-dev # for pkcs11 "ltdl.h"
 		libsqlite3-dev # for "sqlite3.h"
+		libseccomp2  # for "libseccomp.so.2"
```

Comment:
Well, I just mean that it's kind of inconsistent because we don't do it
explicitly for any other ".so" files (only dev headers).